gnovm/stdlibs/std/coins.gno

```diff
@@ -177,8 +177,8 @@ func (cz Coins) Add(b Coins) Coins {
 	return c
 }
 
-// expandNative expands for usage within natively bound functions.
-func (cz Coins) expandNative() (denoms []string, amounts []int64) {
+// ExpandNative expands for usage within natively bound functions.
+func (cz Coins) ExpandNative() (denoms []string, amounts []int64) {
```

Comment:
I would not export these, if anything you can put this in `internal/` so it's visible also in testing stdlibs.

Reply:
- If I put this in `internal`, `internal` will need to import `std.Coins` and  some `std` packages like `banker` need to import `internal` to use so it creates circular imports. 
- If I create `func ExpandNative` and use in `func (cz Coins) expandNative`, we still expose the function

So the simplest way I found is to create a helper in `testing` pkg for that but the ugly thing is I have to duplicate the code.  

Wdyt ?

Reply:
yeah, you're right. just copy the function, but don't export it. it's unnecessary outside of stdlibs